kvbc/src/blockchain_misc.cpp

```diff
@@ -13,11 +13,23 @@
 #include "blockchain_misc.hpp"
 #include "storage/merkle_tree_key_manipulator.h"
 
-namespace concord::kvbc::bcutil {
+namespace concord::kvbc {
 
+block_version BlockVersion::getBlockVersion(const concord::kvbc::RawBlock& raw_block_ser) {
+  return getBlockVersion(raw_block_ser.data(), raw_block_ser.size());
+}
+block_version BlockVersion::getBlockVersion(const std::string_view& raw_block_ser) {
+  return getBlockVersion(raw_block_ser.data(), raw_block_ser.size());
+}
+block_version BlockVersion::getBlockVersion(const char* raw_block_ser, size_t len) {
+  ConcordAssertGE(len, BLOCK_VERSION_SIZE);
+  return *(reinterpret_cast<const block_version*>(raw_block_ser));
```

Comment:
is it safe ? as block_version is an enum , what will be the return value when the block version is not V1